examples/log-generator/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:8.2.1-alpine
+
+WORKDIR /log-generator
+
+ENV ESCRIBA_TIMEOUT 3000
```

Comment:
> It's a default value, doesn't affect the code.

It does. The default value on the code is never used! Imagine someone opens this codebase and wants to change this default. This person may change only the code and it will never work.